src/Combinatorics/Matroids/matroids.jl

```diff
@@ -1019,3 +1019,28 @@ function automorphism_group(M::Matroid)
   resize!(I, nrows(I), length(M))
   return automorphism_group(I; action=:on_cols)
 end
+
+@doc raw"""
+    is_quotient(Q1::Matroid,Q2::Matroid)
+
+Let Q1 and Q2 be matroids on the same groundset with rank $rank(Q1) \leq rank(Q2)$.  
+Check if $Q1$ is a matroid quotient of $Q2$.
+
+# Examples
+
+```jldoctest
+julia> Q1 = uniform_matroid(1,3)
+Matroid of rank 1 on 3 elements
+
+julia> Q2 = uniform_matroid(2,3)
+Matroid of rank 2 on 3 elements
+
+julia> is_quotient(Q1,Q2)
+true
+```
+"""
+function is_quotient(Q1::Matroid,Q2::Matroid)
+    @req matroid_groundset(Q1) == matroid_groundset(Q2) "matroids must be on same groundset" 
+    @req rank(Q1)<=rank(Q2) "matroids must be of equal or increasing rank"
```

Comment:
Again, just wondering, but: why not this (and then also adjust the documentation comment) ?
```suggestion
    rank(Q1) <= rank(Q2) || return false
```
I mean, of course the rank condition is obviously necessary for Q1 to be a quotient of Q2, but we also don't raise an error if one checks `a < b` for two integers when `a` is positive and `b` negative, even though then "obviously" `a < b` is false.

(In contrast, having different groundsets kinda means "those two objects are not really comparable, so throwing an error there makes more sense to me).

Don't get me wrong, this is not a blocker for this PR, perhaps you are following established conventions in matroid theory -- it just seems odd from the outside, that's all :-)

Reply:
Hello, I see what you are saying.

In my opinion, I could be wrong, telling the user that the matroids are not of increasing or equal rank can be informative and and reduces the potential for confusion about the output.

That is, if we have rank(Q1)<rank(Q2), and is_quotient(Q1,Q2) == true, the output is_quotient(Q2,Q1) == false can be misleading about the relationship between these matroids. 

Does this make sense? In any case, I am not attached to either approach.

Reply:
I have no strong reason to argue either way. Unless @bschroter disagrees, I'd say: let's leave it as is for now and if a user complaints about it later, we can still address it.